diff -ur xfce4-eyes-plugin-4.4.1-orig/configure.ac xfce4-eyes-plugin-4.4.1/configure.ac
--- xfce4-eyes-plugin-4.4.1-orig/configure.ac	2010-12-17 17:38:26.000000000 +0100
+++ xfce4-eyes-plugin-4.4.1/configure.ac	2011-03-01 11:48:12.466067636 +0100
@@ -61,7 +61,7 @@
 dnl ***********************************
 XDT_CHECK_PACKAGE([GTHREAD], [gthread-2.0], [2.6.0])
 XDT_CHECK_PACKAGE([GTK], [gtk+-2.0], [2.6.0])
-XDT_CHECK_PACKAGE([LIBXFCEGUI4], [libxfcegui4-1.0], [4.3.90.2])
+XDT_CHECK_PACKAGE([LIBXFCE4UI], [libxfce4ui-1], [4.8.0])
 XDT_CHECK_PACKAGE([LIBXFCE4UTIL], [libxfce4util-1.0], [4.3.90.2])
 XDT_CHECK_PACKAGE([LIBXFCE4PANEL], [libxfce4panel-1.0], [4.3.99.1])
 
diff -ur xfce4-eyes-plugin-4.4.1-orig/panel-plugin/eyes.c xfce4-eyes-plugin-4.4.1/panel-plugin/eyes.c
--- xfce4-eyes-plugin-4.4.1-orig/panel-plugin/eyes.c	2010-12-17 04:48:17.000000000 +0100
+++ xfce4-eyes-plugin-4.4.1/panel-plugin/eyes.c	2011-03-01 11:22:54.644067640 +0100
@@ -33,7 +33,7 @@
 #include <stdlib.h>
 
 #include <libxfce4util/libxfce4util.h>
-#include <libxfcegui4/libxfcegui4.h>
+#include <libxfce4ui/libxfce4ui.h>
 
 #include "eyes.h"
 #include "themes.h"
diff -ur xfce4-eyes-plugin-4.4.1-orig/panel-plugin/eyes.h xfce4-eyes-plugin-4.4.1/panel-plugin/eyes.h
--- xfce4-eyes-plugin-4.4.1-orig/panel-plugin/eyes.h	2010-12-17 04:48:17.000000000 +0100
+++ xfce4-eyes-plugin-4.4.1/panel-plugin/eyes.h	2011-03-01 11:27:42.255067643 +0100
@@ -19,7 +19,7 @@
 #define __EYES_H__
 
 #include <gtk/gtk.h>
-#include <libxfce4panel/xfce-panel-plugin.h>
+#include <libxfce4panel/libxfce4panel.h>
 
 typedef struct
 {
diff -ur xfce4-eyes-plugin-4.4.1-orig/panel-plugin/Makefile.am xfce4-eyes-plugin-4.4.1/panel-plugin/Makefile.am
--- xfce4-eyes-plugin-4.4.1-orig/panel-plugin/Makefile.am	2010-12-17 04:48:17.000000000 +0100
+++ xfce4-eyes-plugin-4.4.1/panel-plugin/Makefile.am	2011-03-01 11:42:08.074067641 +0100
@@ -13,12 +13,12 @@
 	themes.c
 
 xfce4_eyes_plugin_CFLAGS =						\
-	$(LIBXFCEGUI4_CFLAGS)						\
+	$(LIBXFCE4UI_CFLAGS)						\
 	$(LIBXFCE4PANEL_CFLAGS)
 	
 xfce4_eyes_plugin_LDADD =						\
 	$(LIBXFCE4PANEL_LIBS)						\
-	$(LIBXFCEGUI4_LIBS)
+	$(LIBXFCE4UI_LIBS)
 
 desktopdir = $(datadir)/xfce4/panel-plugins
 desktop_in_in_files = eyes.desktop.in.in
